x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -39,22 +39,21 @@ export interface TopAlert {
 
 const NO_INDEX_NAMES: string[] = [];
 const NO_INDEX_PATTERNS: IndexPatternBase[] = [];
+const ALERT_STATUS_REGEX =
+  /\s*and\s*kibana\.alert\.status\s*:\s*".+?"|kibana\.alert\.status\s*:\s*".+?"/gm;
 
 function AlertsPage() {
   const { core, plugins, ObservabilityPageTemplate } = usePluginContext();
+  const [alertFilterStatus, setAlertFilterStatus] = useState(AlertStatus.All);
   const { prepend } = core.http.basePath;
   const refetch = useRef<() => void>();
   const timefilterService = useTimefilterService();
-  const {
-    rangeFrom,
-    setRangeFrom,
-    rangeTo,
-    setRangeTo,
-    kuery,
-    setKuery,
-    workflowStatus,
-    setWorkflowStatus,
-  } = useAlertsPageStateContainer();
+  const { rangeFrom, setRangeFrom, rangeTo, setRangeTo, kuery, setKuery, workflowStatus } =
```

Comment:
@fkanout Do we still need `workflowStatus` here?

Reply:
For this particular use case, not really. But removing it will have a cascade effect in other places (need to remove other props and code util we arrive on something required). Besides, one of the AC is to keep the workflow-related code.